packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -0,0 +1,62 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
```

Comment:
For tests intent, adding comments to use the finalized P0 scenarios summary descriptions.

The other instructions/requirements is contingent on finalizing how exactly are we running the tests. The infrastructure setup and tear down process would be part of it too.
Tracking conversations around this in https://github.com/Azure/azure-sdk-for-js/issues/1915 and related issues.

Reply:
For now, we are not going to add any infrastructure for running these tests. The users will have to create the queues/resources by hand -- I want to capture what someone would need to do today to start running these tests.

Reply:
Ok, added comments for things needed to run the tests.
By users, I'm hoping you mean just the internal team. Because if this is intended to be used as a sample by everyone, that changes things a lot. :)

Reply:
Comments for use in our internal team is fine for now.